components/layout/SideNav.tsx

```diff
@@ -460,22 +467,22 @@ const SideNav = ({ sectionLinks, pages, mobileItems }: SideNavProps) => {
           </StickySectionWrap>
 
           <CategoriesWrap>
-            {keys(categories).map((c) => {
+            {sortCategories(activeSection.href, keys(categories)).map((c) => {
               const categoryDocs = categories[c];
               const groups = groupsFrom(categoryDocs);
 
               return (
                 <Category key={c}>
-                  {c && <CategoryHeading>{c}</CategoryHeading>}
+                  {c !== "undefined" && <CategoryHeading>{c}</CategoryHeading>}
```

Comment:
This is to prevent `"undefined"` from being displayed as a category heading. Not pretty, but didn't want to spend much time on it since I think it'd make sense to do a SideNav cleanup ticket at some point.

Reply:
Sweet! The rouge UNDEFINED category is no more! 

However, not sure if it's related, but the `guides/dwolla.js/index.mdx` file that doesn't have a "group" or "weight" defined is floating to the top of the SideNav. Navigating to that file gives a `Cannot read property 'step' of undefined` error. Any way we could keep it from appearing in the SideNav? 

<img width="328" alt="Screen Shot 2020-05-28 at 11 35 27 PM" src="https://user-images.githubusercontent.com/44006085/83221709-2212f980-a13c-11ea-9aec-1c60cf39a366.png">

Reply:
Yeah, it's doing the same on master (the error). Do you know if there's a category we want the Dwolla.js guide to display under?

Reply:
Yeah, it'd be under the category: FUNDING SOURCES. I believe it already is, but this particular doc doesn't have a group or step defined.

Reply:
I think that's supposed to be a shared index page that's going away until we have some way to handle that situation: https://dwolla.slack.com/archives/GQH89J8G0/p1590505702004800